l10n_br_base/models/res_partner.py

```diff
@@ -182,16 +176,39 @@ def _onchange_zip(self):
             if len(val) == 8:
                 self.zip = "%s-%s" % (val[0:5], val[5:8])
 
-    @api.cr_uid_context
-    def _address_fields(self, cr, uid, context=None):
+    def _address_fields(self):
         """ Returns the list of address fields that are synced from the parent
         when the `use_parent_address` flag is set.
         Extenção para os novos campos do endereço """
-        address_fields = super(ResPartner, self)._address_fields(
-            cr, uid, context=context)
+        address_fields = super(ResPartner, self)._address_fields()
         return list(address_fields + ['l10n_br_city_id', 'number', 'district'])
 
 
+class Bank(models.Model):
```

Comment:
:-1:  this class here is a bad merge: it is already present in https://github.com/OCA/l10n-brazil/blob/595954e3b2442859893250738e5513f03fdca91c/l10n_br_base/models/res_bank.py#L11
Probably before integrating this PR in 8.0 we moved that class into res_bank.py. So this should be removed.